src/gui/SubWindow.cpp

```diff
@@ -26,42 +26,251 @@
 
 #include "SubWindow.h"
 
+#include <QMdiArea>
 #include <QMoveEvent>
 #include <QResizeEvent>
-#include <QWidget>
 
+#include "embed.h"
 
-SubWindow::SubWindow(QWidget *parent, Qt::WindowFlags windowFlags)
-  : QMdiSubWindow(parent, windowFlags)
+
+SubWindow::SubWindow( QWidget *parent, Qt::WindowFlags windowFlags )
+	: QMdiSubWindow( parent, windowFlags )
```

Comment:
Reposting the comment. I'd suggest to move the colons and commas, like this:

```c++
SubWindow::SubWindow(QWidget *parent, Qt::WindowFlags windowFlags) :
    QMdiSubWindow(parent, windowFlags),
    m_buttonSize(17,17),
    m_titleBarHeight(24)
```

That's how this usually looks in the rest of the codebase, unless you have a strong reason for keeping it this way.